packages/app/src/app/pages/Sandbox/Editor/Header/SandboxName/index.tsx

```diff
@@ -90,66 +81,72 @@ export const SandboxName: FunctionComponent = () => {
       (currentSandbox.team ? currentSandbox.team.name : 'My Sandboxes')
     : 'My Sandboxes';
 
-  const spring = useSpring({
```

Comment:
Hmm, why is the animation removed in this case?

Reply:
Ohh, I see, the current animation is quite bad 😅

Reply:
Yeah, it didnt contribute much to the UX. Would be cool to work on a consistent _motion_ design for our interface with danny